CHANGES.md

```diff
@@ -1,5 +1,25 @@
 # Changes
 
+## edge-20.5.5
+
+This edge release adds refinements to the Linkerd multicluster implementation,
+adds new health checks for the tracing add-on, and addresses an issue in which
+outbound requests from the proxy result in looping behavior.
+
+* CLI
+  * Added `allow` and `link` commands, effectively enabling a cluster to have
```

Comment:
these are `multicluster` subcommands so let's qualify them as `multicluster allow` and `multicluster link`.

Reply:
Changed description to introduction of the `multicluster` command and its subcommands. Omitted `allow` and `link` specifically since there are multiple subcommands for `multicluster`.